### ISSUE TITLE
CRD Resource API version deprecated

### ISSUE THREAD

**Issue by Chili-Man** [senthilrch/kube-fledged#66]
When installing the Helm chart, I get the following notice:

```console
apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

**Comment by senthilrch** [comment]
@Chili-Man : Is the helm install successful?

**Comment by Chili-Man** [comment]
Apologies for not leaving more details; The helm install is successful; I'd be more then happy to submit a PR to the CRD up to date with v1, but that will mean that only Kubernetes versions 1.16+ will be supported if I do so. Let me know if that's ok.

**Comment by senthilrch** [comment]
@Chili-Man : Yes that's ok, please update README pre-requisites as well. Raise PR..Thanks

**Comment by senthilrch** [comment]
I'll have this fixed in v0.8.0

**Comment by Chili-Man** [comment]
thanks @senthilrch  !
